Code/Framework/AzFramework/CMakeLists.txt

```diff
@@ -25,13 +26,20 @@ ly_add_target(
             ${pal_dir}
         PRIVATE
             ${common_dir}
+            AZ::AzNetworking
     BUILD_DEPENDENCIES
         PRIVATE
             AZ::AzCore
         PUBLIC
-            AZ::GridMate
+            AZ::AzNetworking
```

Comment:
I have a suggestion of depending on a new API library called AZ::AzNetworking.API which is a HEADERONLY target.
You can use the [ScriptCanvas API ](https://github.com/o3de/o3de/blob/development/Gems/ScriptCanvas/Code/CMakeLists.txt#L93-L107) target as an example

Reply:
I'll look into this.